.github/actions/test/action.yaml

```
@@ -7,5 +7,5 @@ runs:
     run: cargo test --manifest-path tools/slicec-cs/Cargo.toml
     shell: bash
   - name: 🧪 Test
-    run: dotnet test --no-build --verbosity normal --blame-hang-timeout 8m
+    run:  dotnet test --no-build --verbosity normal --blame-hang-timeout 10s tests/IceRpc.Tests/IceRpc.Tests.csproj
```

Is this extra space intentional? It's fine if you want it for some reason. Just double-checking.
```suggestion
    run: dotnet test --no-build --verbosity normal --blame-hang-timeout 10s tests/IceRpc.Tests/IceRpc.Tests.csproj
```

---

Not intentional, just a bad copy and paste.